saltysimulations/salty-747
# feat: fueling and boarding

* Adds a convenient way to load fuel and payload via the CDU
* Adds a feature that saves the amount of fuel, so that the amount of fuel upon starting the next flight is whatever it was when the last flight ended.

![image](https://user-images.githubusercontent.com/26278705/184016290-882098e8-4fce-4bc2-8e0a-b6de61af7c5f.png)
![image](https://user-images.githubusercontent.com/26278705/184016325-641528f0-4859-4fb9-adc5-aaf7bc476d00.png)

**Testing instructions**
* Test the fueling and boarding features extensively, manually inputting values as well as importing from SimBrief, using both kg and lbs units. Make sure the fuel and weight is distributed in a reasonable way. Test out the different rate options. **Note: make sure to use the [new SimBrief airframe](https://www.simbrief.com/system/dispatch.php?sharefleet=141372_1660066082616).**
